polaris.shopify.com/content/whats-new/version-12.md

```diff
@@ -0,0 +1,221 @@
+  - redesign
+  - admin design language
+  - new design
+  - black button
+  - How to migrate to the new design language
+order: 0
+---
+
+# {frontmatter.title}
+
+<Lede>{frontmatter.description}</Lede>
+
+![Illustration of a hand drawing a four pointed star](/images/updates/uplift-beta@2x.png)
+
+---
+
+## Upgrade to v12
+
+Follow our [migration guide](https://polaris.shopify.com/migrating-from-v11-to-v12) to upgrade Polaris from v11 to v12.
+
+## What's changing
+
+### The design language
+
+Polaris version 12 introduces a new design language for Shopify's admin. This includes a style uplift for all of our [components](https://polaris.shopify.com/components), updates to our token values, and a [new web font, Inter](https://polaris.shopify.com/version-guides/migrating-from-v11-to-v12#a-new-web-font).
+
+#### Design principles
+
+The new design language was crafted with guiding principles in mind, ultimately to create surfaces that best serve merchants.
+
+- **Pro feel**: Interactions are highly functional, responsive, and effective.
+- **Meaningful**: Visual language is clear for merchants.
+- **High density**: Space is optimized while maintaining high usability.
+- **Tactility**: Interfaces incorporate a sense of realness.
+- **Commonality**: Objects with similar appearance share a common behavior.
+
+Read more about Polaris' [design principles](https://polaris.shopify.com/design) to start designing in the new language.
+
+### Token refinement
+
+The version 12 updates aim to create an intentional set of tokens that clearly communicates intent so that builders have exactly what they need to apply the new design language on their surfaces. To do this, v12 introduces primitive and semantic token layers.
+
+For more information on how to migrate from v11 tokens, check out the [migration guide](https://polaris.shopify.com/migrating-from-v11-to-v12#token-migrations).
+
+#### Primitive tokens
+
+Primitive tokens are generic keys for the base values of a token scale. Primitive tokens are not context dependent and can be used anywhere in the admin. For example, `--p-space-100` is a primitive space token.
+
+#### Semantic tokens
+
+Semantic tokens are references to base values that are used in specific contexts within the admin. These tokens should never be used for anything other than the concept they’re referencing. When no semantic token is a good fit, a primitive token should be used instead. For example, `--p-space-table-cell-padding` is a semantic token.
+
+#### Updated token resources
+
+| Token guidance                                                                    | Full token list                                                |
+| --------------------------------------------------------------------------------- | -------------------------------------------------------------- |
+| [Color token guidance](https://polaris.shopify.com/design/colors/tokens)          | [Color token list](https://polaris.shopify.com/tokens/color)   |
```

Comment:
We're no longer shipping the new `/design` guidance  with v12 so these L3 tokens pages will not be available for reference and the links will 404. Can we just link to the related `/tokens` pages or is there other content available?

Reply:
```suggestion
| [Color token guidance](https://polaris.shopify.com/design/colors/tokens)          | [Color token list](/tokens/color)   |
```